sdk/appconfiguration/azure-data-appconfiguration/src/test/resources/session-records/getConfigurationSettingNotFound.json

```diff
@@ -1,10 +1,10 @@
 {
```

Comment:
If the test case itself hasn't changed, you don't need to update the records, it adds unnecessary files to the PR.

Reply:
updated